index.html

```diff
@@ -1,234 +1,253 @@
+      <ul>
+        <ul class="marginlistitems">
+          <li class="listitems">Javascript</li>
+          <li class="listitems">Ruby</li>
+          <li class="listitems">Html</li>
+          <li class="listitems">Css</li>
+        </ul>
+        <div class="thumbnail">
+          <div class="rectangle square">
+          </div>
+          <h2 class="headings">Frameworks</h2>
         </div>
-    </footer>
-  </body>
-</html>
+        <ul class="marginlistitems">
+          <li class="listitems">Bootstrap</li>
+          <li class="listitems">Ruby on Rails</li>
+          <li class="listitems">RSPec</li>
+          <li class="listitems">CapyBara</li>
+          <li class="listitems">Selenium</li>
+        </ul>
+        <div class="thumbnail">
+          <div class="rectangle circle">
+          </div>
+          <h2 class="headings ">Skills</h2>
+        </div>
+        <ul class="marginlistitems">
+          <li class="listitems ">Codekit</li>
+          <li class="listitems">Github</li>
+          <li class="listitems">Codepen</li>
+          <li class="listitems">Gitlab</li>
+          <li class="listitems">Terminal</li>
+        </ul>
+      </ul>
+    </section>
+
+    <form action="https://formspree.io/f/xoqzdnpa"
+    method="POST">
+      <section id="getintouch" class="border">
+        <h2 class="intouch">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch</h2>
+        <input class="input border" type="text" placeholder="Full name" maxlength="30" required>
+        <input class="input border" type="email" placeholder="Email address" required>
+        <textarea class="input" name="textarea" id="textarea" cols="30" rows="5" placeholder="Enter text here"
+          maxlength="500" required></textarea>
+          <div class="centerbtn"><button type="submit" class="getintouchbtn work-button">Get in touch</button>
+        </div>
+      </section>
+    </form>
+  
+  </main>
+
+  <footer>
+    <div class="sociallinks">
+      <ul class="footercenter">
+        <li>
+          <a href="https://github.com/fpsapc" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-github social"></i></a>
+        </li>
+        <li>
+          <a href="https://linkedin.com/in/salman-ahmad1987" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-linkedin social"></i></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/company/vivimedia/about/" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-square-facebook social"></i></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/salman-ahmadkhan1987" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-twitter social"></i></a>
+        </li>
+        <li>
+          <a href="mailto:fpsapc@gmail.com" target="_blank" rel="noopener"> <i
+              class="fa-sharp fa-solid fa-m social"></i></a>
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] Kindly consider using the social icons from the figma design as sticking to the figma design as much as possible is one of the project requirements.kindly consider downloading the figma design social icons to use.
figma design
![fd](https://user-images.githubusercontent.com/96683843/220425008-fef1f15e-a185-4a67-aed4-af871b4fb591.jpg)

your design
![as](https://user-images.githubusercontent.com/96683843/220425029-a4a52edf-25d3-46fd-941c-b44e0e87206d.jpg)